src/libraries/System.Diagnostics.Tracing/ref/System.Diagnostics.Tracing.cs

```diff
@@ -14,6 +14,11 @@ public enum EventActivityOptions
         Recursive = 4,
         Detachable = 8,
     }
+    [AttributeUsage(System.AttributeTargets.Class)]
+    public sealed class EventSourceEventGenerateAttribute : Attribute
```

Comment:
Should we call it `GeneratedEventSourceEventAttribute`, akin to `GeneratedRegex`?

Reply:
Or `GeneratedTracingEventAttribute`